modules/gltf_blend/editor_scene_importer_blend.cpp

```diff
@@ -0,0 +1,277 @@
+		parameters_arg += "export_force_sampling=True,";
+	} else {
+		parameters_arg += "export_force_sampling=False,";
+	}
+	if (p_options.has(SNAME("blender/meshes/export_bones_deforming_mesh_only")) && p_options[SNAME("blender/meshes/export_bones_deforming_mesh_only")]) {
+		parameters_arg += "export_def_bones=True,";
+	} else {
+		parameters_arg += "export_def_bones=False,";
+	}
+	if (p_options.has(SNAME("blender/nodes/modifiers")) && p_options[SNAME("blender/nodes/modifiers")]) {
+		parameters_arg += "export_apply=True";
+	} else {
+		parameters_arg += "export_apply=False";
+	}
+
+	String unpack_all;
+	if (p_options.has(SNAME("blender/materials/unpack_enabled")) && p_options[SNAME("blender/materials/unpack_enabled")]) {
+		unpack_all = "bpy.ops.file.unpack_all(method='USE_LOCAL');";
+	}
+	String common_args = vformat("filepath='%s',", parameters["sink"]) +
+			"export_format='GLTF_SEPARATE',"
+			"export_yup=True," +
+			parameters_arg;
+	String scene_to_blend_setting = "import bpy, os, sys;"
+									"print('Blender is 3.0 or higher is required.', file=sys.stderr) if bpy.app.version < (3, 0, 0) else None;" +
+			vformat("bpy.ops.wm.open_mainfile(filepath='%s');", parameters["source"]) +
+			unpack_all +
+			"bpy.ops.export_scene.gltf(export_keep_originals=True," + common_args + ");";
+	String script = scene_to_blend_setting;
+	List<String> args;
+	args.push_back("--background");
+	args.push_back("--python-expr");
+	args.push_back(script);
+	print_verbose(script);
+	String standard_out;
+	String standard_err;
+	int32_t ret = OS::get_singleton()->execute(addon_path, args,
+			&standard_out, &ret, true);
+	print_verbose(standard_out);
+	if (ret != OK) {
+		*r_err = ERR_SCRIPT_FAILED;
+		print_error("Blender returned " + itos(ret));
+		return nullptr;
+	}
+	// Used GLTFDocument instead of gltf importer to keep image references
+	Ref<GLTFDocument> gltf;
+	gltf.instantiate();
+	List<String> deps;
+	Ref<GLTFState> state;
+	state.instantiate();
+	String base_dir;
+	if (p_options.has(SNAME("blender/materials/unpack_enabled")) && p_options[SNAME("blender/materials/unpack_enabled")]) {
+		base_dir = sink.get_base_dir();
+	}
+	err = gltf->append_from_file(sink.get_basename() + ".gltf", state, p_flags, p_bake_fps, base_dir);
+	if (err != OK) {
+		if (r_err) {
+			*r_err = FAILED;
+		}
+		return nullptr;
+	}
+	return gltf->generate_scene(state, p_bake_fps);
+}
+
+Ref<Animation> EditorSceneFormatImporterBlend::import_animation(
+		const String &p_path, uint32_t p_flags, const Map<StringName, Variant> &p_options,
+		int p_bake_fps) {
+	return Ref<Animation>();
+}
+
+Variant EditorSceneFormatImporterBlend::get_option_visibility(
+		const String &p_path, const String &p_option,
+		const Map<StringName, Variant> &p_options) {
+	if (p_option.begins_with("animation/")) {
+		if (p_option != "animation/import" &&
+				!bool(p_options["animation/import"])) {
```

Comment:
I think your IDE hard wraps a bit too aggressively. We don't have a max line length and wrapping a relatively short condition like this doesn't really make things more readable IMO.

Not a big deal though.